charts/observability-pipelines-worker/ci/all-values.yaml

```diff
@@ -1,3 +1,9 @@
+args:
+  - run
+  - --skip-key-validation
```

Comment:
Note, key validation work is still blocked on the OPW side -- if changes occur, this will need updating.

Reply:
Changes have been merged on the OPW side.